edubots_surveys.Rnw

```diff
@@ -120,6 +120,11 @@
   addressref={aff1},
   email={fbarranco@ugr.es}
 ]{\inits{F.}\fnm{Francisco} \snm{Barranco}}
+\author[
+  addressref={aff2},
+  email={n.h.abbas@leeds.ac.uk}
```

Comment:
You will need to add also your address, a little down below, from line 144. Check how it's done for the rest; this is a cross-reference to the addresses defined below. If you put it here answering to me, I can do it for you if you want.